tests/interfaces.rs

```diff
@@ -98,28 +98,27 @@ async fn interfaces() -> anyhow::Result<()> {
         rust_blobstore_fs,
         rust_blobstore_s3,
         rust_http_client,
-        rust_http_server,
         rust_keyvalue_redis,
         rust_keyvalue_vault,
-        rust_messaging_nats,
     ) = join!(
         providers::rust_blobstore_fs(),
         providers::rust_blobstore_s3(),
         providers::rust_http_client(),
-        providers::rust_http_server(),
         providers::rust_keyvalue_redis(),
         providers::rust_keyvalue_vault(),
-        providers::rust_messaging_nats(),
```

Comment:
I'm glad we are testing the builtins as well, but I am concerned with removing the NATS messaging provider from the regular test suite. I suppose little changes here and `wasi:messaging` is a likely successor, but I would like to keep it in the regular integration test.